src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -0,0 +1,400 @@
+﻿using System;
```

Comment:
I assume this is a split off part from the microbe editor? If so you should edit the new separate patch map tab scene so that it uses this code and make sure there weren't any new additions in there that your version hasn't picked up.